src/ert/storage/local_experiment.py

```diff
@@ -253,6 +257,10 @@ def parameter_info(self) -> dict[str, Any]:
     @property
     def response_info(self) -> dict[str, Any]:
         info: dict[str, Any]
+        if not (self.mount_point / self._responses_file).exists():
```

Comment:
This was just changed to give a `FileNotFoundError` because the exception did not show full path. Yes the `responses.json` does not exist, but the entire directory might be wrong:
https://github.com/equinor/ert/commit/76a2ba9f5b61c9d4bf08b42adde4edb2b0c62271